packages/atlas-react/tests/components/BasicButton.test.js

```diff
@@ -1,5 +1,6 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render } from "@testing-library/react";
+import { screen } from "shadow-dom-testing-library";
```

Comment:
@kvuzet looks like there is a way, by using [this library](https://www.npmjs.com/package/shadow-dom-testing-library), to use the shadow DOM that is generated by the WC, which is better than mocking the WC for tests.